src/Skin.js

```diff
@@ -171,6 +171,46 @@ class Skin extends EventEmitter {
      */
     updateSilhouette () {}
 
+    /**
+     * Set the contents of this skin to an empty skin.
+     * @param {ImageData|HTMLImageElement|HTMLCanvasElement|HTMLVideoElement} bitmapData - new contents for this skin.
+     * @param {int} [costumeResolution=1] - The resolution to use for this bitmap.
+     * @param {Array<number>} [rotationCenter] - Optional rotation center for the bitmap. If not supplied, it will be
```

Comment:
we're not passing any parameters, so they shouldn't be in the documentation

Reply:
Ooh good catch, thanks!